src/Logger/RulesLog.php

```diff
@@ -72,6 +73,10 @@ public function log($level, $message, array $context = array()) {
       'timestamp' => $context['timestamp'],
     );
 
+    $logger_entry = new LoggerEntry();
+    $logger_entry->log($level, $message, $context, $this->parser);
```

Comment:
The logger entry should not have a log() method. It should have a setMessage() method, a setLevel() method etc.